usr/bin/hwctl

```diff
@@ -43,7 +43,7 @@ function system-info {
 	# Playtron package versions
 	AUTOTEST_VERSION=$(rpm -qa | grep autotest | cut -c10- | sed 's/.[a-z].*//')
 	PLAYSERVE_VERSION=$(rpm -qa | grep playserve | cut -c11- | sed 's/.[a-z].*//')
-	PLAYTRON_APP_VERSION=$(rpm -qa | grep playtron-app | cut -c14- | sed 's/.[a-z].*//')
+	GRID_VERSION=$(rpm -qa | grep grid | cut -c6- | sed 's/.[a-z].*//')
```

Comment:
The word "grid" is very generic. A total of 124 packages in Fedora contain that word.  None (besides our own Grid) are used in PlaytronOS but it could become an issue for us one day and if users are installing the Grid RPM on a different RPM-based Linux distribution.

```
$ sudo dnf search grid | cut -d. -f1 | grep grid | wc -l
124
```

Perhaps we can make the check more specific. In the example below, it must start with "grid" and then look for the period that exists before the CPU architecture is specified:

```
rpm -qa | grep -P "^grid\."
```

Reply:
Actually, to simply this even more, we can just query the exact package name with `rpm -q grid`